### PR TITLE
Added in interactive programming resources for Go, Kotlin, and added in an R section 

### PR DESCRIPTION
**Pull request by kree666**
## What does this PR do?
Adds in interactive programming resources for several different programming languages including Go, Kotlin, and added in an R section 

### Description
These resources provide free interactive programming opportunities in order to learn Go, Kotlin, and R.

### Why is this valuable (or not)?
Provides people with an opportunity to learn these languages for free

### How do we know it's really free?
No payment is needed

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -303,6 +306,11 @@
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
 
 
+## R
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:309]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

:information_source: In this case, go ahead

File: more/free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -138,6 +139,7 @@
 
 ### Go
 
+* [Go at Codecademy](https://www.codecademy.com/learn/learn-go)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:142]: Right title, instructor attributions, platform and signup notes are always the best contribution. Follow format

```suggestion
* [Learn Go](https://www.codecademy.com/learn/learn-go) - Google, Robert Griesemer, Rob Pike, and Ken Thompson (Codecademy) *(account required)*
```

File: more/free-programming-interactive-tutorials-en.md [hunk 3]
```diff
@@ -222,6 +224,7 @@
 
 ### Kotlin
 
+* [Codecademy Kotlin Course](https://www.codecademy.com/learn/learn-kotlin)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:227]: Needs follow suggested guidelines

File: more/free-programming-interactive-tutorials-en.md [hunk 4]
```diff
@@ -303,6 +306,11 @@
 * [Python Tutorial](https://www.w3schools.com/python) - W3Schools
 
 
+## R
+
+* [Codecademy R course](https://www.codecademy.com/learn/learn-r)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:311]: Needs follow suggested guidelines

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to make some changes, When done, please request for a rewiew again using :arrows_clockwise: button.

:warning: I doubt if this needs to be moved to courses but is also a fact that the most of Codecademy's are here, as interactive.

**Comment by eshellman** [comment]
R is worth a section.

**Pull request merged**